package.json

```diff
@@ -88,15 +89,16 @@
     "lint:styles": "stylelint \"{projects,feature-libs}/**/*.scss\"",
     "prettier": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\"",
     "prettier:fix": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\" --write",
-    "start": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx serve storefrontapp --configuration=development",
+     
+    "start": "env-cmd --no-override -e dev,b2b,cpq,$SPA_ENV nx serve storefrontapp --configuration=development",
```

Comment:
should cpq really be used in the default start command?

Reply:
Hi Radhep,
 To run cpq-quote, we'll need to include cpq in the npm start command

Reply:
Can you create a new command for this?

Reply:
Hi Radheep, I added this for testing purposes; I have reverted the code now.